build.gradle

```diff
@@ -58,11 +53,12 @@ task funcTest(type: Test) {
     testClassesDirs = sourceSets.funcTest.output.classesDirs
     classpath = sourceSets.funcTest.runtimeClasspath
     jvmArgs '-Xmx1g'
+    systemProperty('ignoreDeprecations', 'true')
```

Comment:
You should not need it anymore, unless it occurs in your functional tests for android. I tried to fix all warning on my side in the last release.

Reply:
Android Gradle Plugin (latest stable version, 3.3.0) still uses `DefaultTask.newInputFile()`, `ProjectLayout.fileProperty()`.